Makefile

```diff
@@ -33,7 +35,19 @@ copy-rpms-to-test:
 	- rm test/testing-rpms/*.rpm
 	find `pwd`/output -name '*.rpm' -printf '%f\n' -exec cp {} test/testing-rpms/ \;
 
-vagrant-test: rpm copy-rpms-to-test
+check-working-directory:
+	@if [ "`git status --porcelain --untracked-files=no | wc -l`" != "0" ]; then \
+	  echo "Uncommited changes, refusing (Use git add . && git commit or git stash to clean your working directory)."; \
+	  exit 1; \
+	fi
+
+vagrant-test: check-working-directory rpm copy-rpms-to-test
```

Comment:
Should copy-rpms-to-test depend on rpm, rather than vagrant-test depending on both?